packages/patternfly-4/react-virtualized-extension/src/components/Virtualized/VirtualizedTable.test.tsx

```diff
@@ -0,0 +1,203 @@
+import * as React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
I think you can remove this `eslint-disable`, you use `clsx` below so the linter is probably not calling it an extraneous dependency anymore